pom.xml

```diff
@@ -283,6 +283,7 @@
       Overridable test home. So that you can call individual pom files directly without
       things breaking.
     -->
+    <session.executionRootDirectory>/tmp</session.executionRootDirectory>
```

Comment:
OUt of curiosity why did we need this?

Reply:
I added it during local testing and forgot to roll back.